test/api_integration/apis/status/status.js

```diff
@@ -26,7 +25,7 @@ export default function ({ getService }) {
           expect(body.version.build_number).to.be.a('number');
 
           expect(body.status.overall).to.be.an('object');
-          expect(body.status.overall.level).to.be('available');
+          expect(body.status.overall.level).to.be.a('string');
```

Comment:
This test is here to ensure the shape of the data. Asserting against the actual status was causing flakiness if for any reason Kibana is not fully green when the test is run